### PR TITLE
VT Updates

### PR DESCRIPTION
**Pull request by FishheadNate**
- the original all.json file was named and was missing subunit labels.
- created wildlife_units.json to replace all.json and have the correct unit & subunit labels
- wilidlife_areas.json is new, but needs labels added to the individual areas: https://github.com/OpenBounds/OpenHuntingData/issues/361

### CODE REVIEW

File: sources/US/VT/wildlife-areas.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+   "name" : "VT Wildlife Management Areas",
+   "url": "https://data.openbounds.org.s3.amazonaws.com/USAHunting/manual-downloads/vt-wildlife-management-areas.zip",
+   "species": [
+      "all"
+   ],
+   "attribution": "Vermont Fish and Wildlife Department",
+   "properties": {
+      "id": "WMA_name",
+      "name": "WMA_name",
```
Comment (JesseCrocker) [inline sources/US/VT/wildlife-areas.json]: Comma on this line is making this file invalid

**Comment by JesseCrocker** [comment]
Error:
```
Processing sources/US/VT/wildlife-units.json
Downloading https://data.openbounds.org.s3.amazonaws.com/USAHunting/manual-downloads/vt-wildlifle-management-untis.zip
INFO:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): data.openbounds.org.s3.amazonaws.com
/usr/local/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:789: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.org/en/latest/security.html
  InsecureRequestWarning)
Reading vt-wildlifle-management-untis.zip
Traceback (most recent call last):
  File "./Processing/process.py", line 78, in process
    source_filename=source.get("filenameInZip", None))
  File "/Users/jesse/projects/OpenBounds/OpenHuntingData/Processing/adapters/shp.py", line 29, in read
    raise Exception("Found multiple shapefiles in zipfile")
Exception: Found multiple shapefiles in zipfile
Found multiple shapefiles in zipfile
Error processing file sources/US/VT/wildlife-units.json

Failed sources: sources/US/VT/wildlife-units.json
```

**Comment by FishheadNate** [comment]
Corrected json file format

**Comment by JesseCrocker** [comment]
Still an error with wildlife-units.json
```
Processing sources/US/VT/wildlife-units.json
Downloading https://data.openbounds.org.s3.amazonaws.com/USAHunting/manual-downloads/vt-wildlifle-management-untis.zip
Returning https://data.openbounds.org.s3.amazonaws.com/USAHunting/manual-downloads/vt-wildlifle-management-untis.zip from local cache
Reading vt-wildlifle-management-untis.zip
Traceback (most recent call last):
  File "./Processing/process.py", line 78, in process
    source_filename=source.get("filenameInZip", None))
  File "/Users/jesse/projects/OpenBounds/OpenHuntingData/Processing/adapters/shp.py", line 29, in read
    raise Exception("Found multiple shapefiles in zipfile")
Exception: Found multiple shapefiles in zipfile
Found multiple shapefiles in zipfile
Error processing file sources/US/VT/wildlife-units.json

Failed sources: sources/US/VT/wildlife-units.json
```

It needs a `filenameInZip` entry

**Comment by FishheadNate** [comment]
I'll make sure the filenames in the zip folder match the name of the zip folder from here on out

**Comment by JesseCrocker** [comment]
When a zip file contains multiple .shp  files, the JSON file needs to have the key `filenameInZip` which sets which file from the zip file to use.

**Pull request merged**